static/js/components/CommentForms.js

```diff
@@ -65,59 +66,77 @@ const commentForm = (
   isComment: boolean,
   disabled: boolean,
   autoFocus: boolean,
-  wysiwyg: boolean = false
-) => (
-  <div className="reply-form">
-    <form
-      onSubmit={onSubmit}
-      className="form"
-      onKeyDown={e => {
-        if (e.key === "Enter" && e.ctrlKey && !disabled && !isEmptyText(text)) {
-          onSubmit(e)
+  wysiwyg: boolean = false,
+  onTogglePopup?: Function
+) => {
+  return (
+    <div className="reply-form">
+      <form
+        onSubmit={
+          userIsAnonymous() && onTogglePopup
+            ? R.compose(
+              onTogglePopup,
+              e => e.preventDefault()
+            )
```

Comment:
we have a little helper called `preventDefaultAndInvoke`, can you use that here instead of the `R.compose` setup?